data-types/ints_and_floats/tests/test_ints_and_floats.py

```diff
@@ -1,80 +1,74 @@
 import pytest
-
+import math
 
 class IntsAndFloatsTest:
     def test_one(self):
         lucky = 7
         unlucky = 13
         # Using the two variables defined above,
         # add the lucky number and the unlucky number
-        sum = None
+        sum = lucky + unlucky
         assert sum == 20
 
-    @pytest.mark.skip
     def test_two(self):
         lucky = 7
         unlucky = 13
         # Using the two variables defined above,
         # subtract the unlucky from the lucky
-        difference = None
+        difference = lucky - unlucky
         assert difference == -6
 
-    @pytest.mark.skip
     def test_three(self):
         lucky = 7
         unlucky = 13
         # Using the two variables defined above,
         # divide unlucky by lucky
         # NOTE: this is integer division
-        quotient = None
+        quotient = unlucky//lucky
         assert quotient == 1
 
-    @pytest.mark.skip
     def test_four(self):
         lucky = 7
         unlucky = 13
         # Using the two variables defined above,
         # divide unlucky by lucky
-        quotient = None
+        quotient = unlucky/lucky
         assert quotient == 1.8571428571428572
 
-    @pytest.mark.skip
     def test_five(self):
         lucky = 7
         unlucky = 13
         # Using the two variables defined above,
         # find the remainder of the unlucky divided by the lucky
-        remainder = None
+        remainder = unlucky%lucky
         assert remainder == 6
 
-    @pytest.mark.skip
     def test_six(self):
         lucky = 7
         # Using the variable defined above,
         # find out if the lucky number is even
-        even = None
-        assert even == False
+        even = ""
+        if (7 % 2) == 0:
```

Comment:
Couple things here. Because conditionals return booleans you can set your `even` variable to the value of condition. Also, assertions against booleans should use the `is` keyword instead of `==`
```suggestion
        even = (lucky % 2)
        assert even is False
```